app/src/main/res/layout/view_clinical_decision_support_bp_high.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:id="@+id/rootView"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:layout_marginHorizontal="@dimen/spacing_8"
+  android:layout_marginTop="@dimen/spacing_8"
+  android:layout_marginBottom="@dimen/spacing_0"
+  android:background="@drawable/background_clinical_decision_alert"
+  android:elevation="@dimen/spacing_4"
+  android:padding="@dimen/spacing_8"
+  android:visibility="gone"
+  tools:visibility="visible">
+
+  <ImageView
+    android:id="@+id/clinicalDecisionAlertIcon"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:contentDescription="@null"
+    app:layout_constraintBottom_toBottomOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    app:srcCompat="@drawable/ic_clinical_decision_warning" />
+
+  <com.google.android.material.textview.MaterialTextView
+    android:id="@+id/clinicalDecisionAlertTitle"
+    android:layout_width="@dimen/spacing_0"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="@dimen/spacing_8"
+    android:text="@string/clinical_decision_support_alert_bp_high_title"
+    android:textAppearance="?attr/textAppearanceSubtitle1Medium"
```

Comment:
We should add margin horizontal for the texts here. As it's in the design as well.